.github/workflows/ci.yml

```diff
@@ -15,7 +15,7 @@ jobs:
         rust: [beta, stable, 1.36.0]
         include:
           - rust: nightly
-            rustflags: --cfg thiserror_nightly_testing
+            rustflags: --cfg stdbacktrace
```

Comment:
I think this needs to be tweaked to run this correctly in CI. `stdbacktrace` is a feature, while `thiserror_nightly_testing` wasn't.